src/index.js

```diff
@@ -126,6 +128,10 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
     render() {
       let { pastDelay, error, Component } = this.state;
 
+      if (!isWebpack && serverSideRequirePath) {
```

Comment:
@smashercosmo you could always wrap loadable and implement your own flushServerSideRequires, I did that [here](https://github.com/60frames/react-boilerplate/blob/master/src/components/lib/loadable/loadable.js) as I also bundle the server side code with webpack so needed the module ids rather than an absolute path.